nj/conf.py

```diff
@@ -0,0 +1,15 @@
+"""conf"""
```

Comment:
yes, that'd be ideal!
TBH, I made theses changes so it could quickly work for my purposes.
but more time needed to make it proper:
Proper config, some refactoring (the conditions I added are not easy to read, just patching),...